include/Common.h

```diff
@@ -172,29 +172,45 @@ typedef enum {
 
 
 typedef enum {
-  // storage commands -> key_va -> success_or_failure
```

Comment:
Would you please move these non-gtest related code into a separate commit?

Reply:
okay

Reply:
@mckelvin non-gtest related code will put in next pr。